Neos.Fusion/Classes/Core/ObjectTreeParser/FilePatternResolver.php

```diff
@@ -56,13 +57,14 @@ class FilePatternResolver
      * @return array|string[]
      * @throws Fusion\Exception
      */
-    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves = null, string $defaultFileEndForUnspecificGlobbing = '.fusion'): array
+    public static function resolveFilesByPattern(string $filePattern, ?string $filePathForRelativeResolves, string $defaultFileEndForUnspecificGlobbing): array
     {
-        $filePattern = trim($filePattern);
+        $filePattern = Files::getUnixStylePath(trim($filePattern));
         if ($filePattern === '') {
             throw new Fusion\Exception("cannot resolve empty pattern: '$filePattern'", 1636144288);
         }
-        if ($filePattern[0] === '/') {
+        $isAbsoluteWindowsPath = str_contains($filePattern, ':') && preg_match('`^[a-zA-Z]:/[^/]`', $filePattern) === 1;
+        if ($filePattern[0] === '/' || $isAbsoluteWindowsPath) {
```

Comment:
Maybe a new method `isAbsolutePath` in `Files` could cover this logic?

Reply:
was thinking, the same but it lives in flow https://github.com/neos/flow-development-collection/blob/8.3/Neos.Utility.Files/Classes/Files.php and that would be cruel ^^

Reply:
> and that would be cruel ^^

What would be cruel? And why?

Reply:
because we dont specify the exact flow patch wed need 

https://github.com/neos/neos/blob/851c073c49c0d9e85f7081cc939c159401286343/composer.json#L11

https://github.com/neos/fusion/blob/5b45a833a1d8e900bbe04d18b31aed37c98aa7e6/composer.json#L10

so updating fusion alone would work (by composer constraints) but would fail at runtime because flow isnt at the latest patch ... then again who does this ...

Reply:
Then you could require Flow in a matching minimum version…